packages/js/src/feeds/feeds.ts

```diff
@@ -1,33 +1,187 @@
-import type { PaginatedResponse } from '../types';
-import { BaseModule } from '../utils/base-module';
+import { BaseModule } from '../base-module';
 import { Notification } from './notification';
-
-interface FetchFeedOptions {
-  page?: number;
-  feedIdentifier?: string | string[];
-  seen?: boolean;
-  read?: boolean;
-  limit?: number;
-  payload?: Record<string, unknown>;
-}
+import { PaginatedResponse, NotificationStatus, TODO } from '../types';
+import type {
+  FetchFeedArgs,
+  FetchCountArgs,
+  MarkNotificationAsArgs,
+  MarkAllNotificationsAsArgs,
+  RemoveNotificationArgs,
+  RemoveAllNotificationsArgs,
+  MarkNotificationActionAsArgs,
+  MarkNotificationsAsArgs,
+  RemoveNotificationsArgs,
+  MarkNotificationAsByIdArgs,
+  MarkNotificationAsByInstanceArgs,
+  MarkNotificationsAsByIdsArgs,
+  MarkNotificationsAsByInstancesArgs,
+  RemoveNotificationByIdArgs,
+  RemoveNotificationByInstanceArgs,
+  RemoveNotificationsByIdsArgs,
+  RemoveNotificationsByInstancesArgs,
+  MarkNotificationActionAsByIdArgs,
+  MarkNotificationActionAsByInstanceArgs,
+} from './types';
+import { READ_OR_UNREAD, SEEN_OR_UNSEEN } from '../utils/notification-utils';
+import { markActionAs, markNotificationAs, markNotificationsAs, remove, removeNotifications } from './helpers';
 
 export class Feeds extends BaseModule {
-  async fetch({ page = 1, ...restOptions }: FetchFeedOptions): Promise<PaginatedResponse<Notification>> {
+  async fetch({ page = 0, status, ...restOptions }: FetchFeedArgs = {}): Promise<PaginatedResponse<Notification>> {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** I think we should start page numbers from 1
and handle internally by page = --page;‏

Reply:
yeah, everywhere in our APIs we are using a page from 0, and in sdks as well :(